changes/191b3876553873b7a8ad0e2e61a43dd7.yaml

```diff
@@ -0,0 +1,7 @@
+---
+desc: |
+  The base Cell now captures runtime information for its listening HTTPS host and port information. This runtime
```

Comment:
Mentioning class names seems a bit over-engineering-ish. KISS: "Update getCellInfo() API to include HTTPS listener host and port."